assignments/prototypes.js

```diff
@@ -32,6 +51,20 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+// 
+function Humanoid(objName)  {
+  GameObject.call(this, objName);
```

Comment:
>GameObject.call(this, objName);
  CharacterStats.call(this, objName);

You only need one of these. Which one do you think it is and why?